.prettierrc

```diff
@@ -1,6 +1,6 @@
 {
   "arrowParens": "avoid",
-  "printWidth": 100,
+  "printWidth": 90,
```

Comment:
Prettier changed the behavior of method chaining linebreaks. Before the upgrade, it would not re-format user specified linebreaks in method chains. After the upgrade it uses printWidth to determine when to break method chains. 100 character long method chains are very hard to read. I adjusted printWidth down to try to strike a balance.